dipy/workflows/segment.py

```diff
@@ -271,9 +272,9 @@ def run(self, streamline_files, model_bundle_files,
                 logging.info("Bundle adjacency Metric {0}".format(ba))
                 logging.info("Bundle Min Distance Metric {0}".format(bmd))
 
-            new_tractogram = nib.streamlines.Tractogram(recognized_bundle,
+            new_tractogram = StatefulTractogram(recognized_bundle,
```

Comment:
missing `reference`

Reply:
The reference will be a trk filename I believe? Also I couldn't figure that out, so any help will be appreciated

Reply:
Hi @sidkapoor97,

You can refer to this [RecoBundles example in DIPY](https://dipy.org/documentation/1.1.1./examples_built/bundle_extraction/#example-bundle-extraction).